defaults/main.yml

```diff
@@ -116,3 +121,16 @@ archivematica_src_automationtools_transfers_pidfile: "/var/archivematica/automat
 # Acceptance tests
 archivematica_src_acceptance_tests_browser_list: [ "Chrome" , "Firefox" ]
 archivematica_src_acceptance_tests_chromedriver_version: "2.29"
+
+# Configure AM
+# archivematica_src_configure_am_api_key & archivematica_src_configure_ss_api_key can be defined as vars. If not defined they will be autogenerated by the playbook
+archivematica_src_configure_ss_user: "test"                  # SS superuser
+archivematica_src_configure_ss_password: "test"              # SS superuser password
```

Comment:
I think passwords should be something line "PleaseChangeMe!", so users don't forget to set them